plotly/offline/offline.py

```diff
@@ -8,40 +8,29 @@
 import json
 import os
 import uuid
-
-import requests
+from pkg_resources import resource_string
+import warnings
```

Comment:
:cow2: mind flipping these import statements? We typically do `from ...` after `import ...`.